Prepare to publish (#3)
fingerprintjs/action-semantic-release-info: README.md

# GitHub Action - Get Next Semantic Release Info

> This GitHub Action gets next semantic release info, does not publish. export the info as output variables.

Reviewer: ```suggestion
This GitHub Action summarizes information about the next semantic release. It does not publish the release. The information is exported as output variables of the GitHub action.
```

> **Note**
> This package isn’t part of our core product. It’s kindly shared “as-is” without any guaranteed level of support from Fingerprint. We warmly welcome community contributions.

## Usage

> Add this step in your workflow file

Reviewer: ```suggestion
Add this step to your GitHub workflow file: 
```

> In this example I get changelog for the future release and add a comment to the pr with it.  

Reviewer: GitHub doesn't let me comment on the correct line but in the previous section, the ending should be the beginning: 

## Output variables

After the action is completed, you can access the output variables using this pattern: 

```
${{ steps.semantic_release_info.outputs.<variable name> }}
```

- list of variables....

> In this example I get changelog for the future release and add a comment to the pr with it.  

Reviewer: ```suggestion
For example, you can get a changelog of a future release and add it as a comment to a pull request: 
```
